vsintegration/fsharp-vsintegration-unittests-build.proj

```diff
@@ -31,6 +31,7 @@
   <Import Project="..\src\root.traversal.targets"/>
 
   <Target Name="Restore">
+    <MSBuild Projects="@(ProjectFiles)" Targets="Restore" />
```

Comment:
If you're going to restore everything then remove all of the `<SdkProjects>` lines; those only existed to restore a subset of the projects.

Reply:
agreed we don't need this